src/main/java/com/google/alpollo/model/SongEntity.java

```diff
@@ -9,12 +9,21 @@
  */
 public class SongEntity {
   private final String name;
-  // Each entity has a salience score, telling us how important the word is in the given context.
+  /**  Each entity has a salience score, telling us how important the word is in the given context. */
   private final double salience;
+  /** 
+   * The API sets each entity to a category.
+   * e.g. PERSON, LOCATION, WORK_OF_ART
+   */ 
+  private final String type;
```

Comment:
How do we get this string (i.e. "PERSON", "LOCATION"), etc? Is it straight from the API? If it is, we need to make it explicit in the docs, i.e. say it matches the entities from _____ API and preferably include a link pointing to all entities available.

If instead you know in advance all the possible types and are setting them manually, maybe we should use an `enum` here instead of a string.